### ISSUE TITLE
lv_group_remove_obj issue

### ISSUE THREAD

**Issue by Moo2017** [lvgl/lvgl#869]
Hi All,
I've upgraded my lvlg from v5.1 to v5.3 . Now I have issue with lv_group_remove_obj . when I delete and object and run lv_obj_del system crashes and I figure out "break" statement in lv_group_remove_obj causes the issue . When I remove "break" statement it works fine rather than that it crashes. I am guessing this happens when you have an object and then you add another two objects . If you remove the parent of the last two added objects this happens. I should mention that I am using external keypad so I add objects to the group.

**Comment by kisvegabor** [comment]
Hi,

Please share a very minimal code to reproduce the issue.

**Comment by Moo2017** [comment]
Hi @kisvegabor 
I found what causes the issue. I created a btn and added to a group, then to create the second btn, I copied the previous btn ( second_btn = lv_btn_create(parent, prev_btn) ) , and then added the new btn to the group while since this had been copied from previous btn it had been already added to the group . 
However I'av attached the code here:
[ui_test.txt](https://github.com/littlevgl/lvgl/files/2901662/ui_test.txt)

In the above code , test_remmove_grp is where application is started. 
If you first click on "Network Config" and then hit back icon to exit from this menu and then hit "Device Config" system is crashes.
